src/pages/ParkJuHee/Login/Login.jsx

```diff
@@ -0,0 +1,111 @@
+import React, { useState } from 'react';
+import './Login.scss';
+import { Link, useNavigate } from 'react-router-dom';
+
+const Login = () => {
+  const navigate = useNavigate();
+  const [email, setEmail] = useState('');
+  const [password, setPassword] = useState('');
+  const [isValid, setIsValid] = useState(false);
+
+  const handleEmail = (e) => {
+    setEmail(e.target.value);
+    setIsValid(isValidEmail(e.target.value));
+  };
```

Comment:
아래처럼 구조분해할당 할 수 있습니다
```suggestion
  const handleEmail = (e) => {
  const {value}=e.target
    setEmail(value);
    setIsValid(isValidEmail(value));
  };
```